packages/postgres-database/src/simcore_postgres_database/migration/versions/cf3bac482ce0_add_file_meta_data_deprecation_date.py

```diff
@@ -0,0 +1,33 @@
+"""add services_meta_data deprecation date
+
+Revision ID: cf3bac482ce0
+Revises: 90ed05df20d7
+Create Date: 2022-06-21 14:43:57.915189+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "cf3bac482ce0"
+down_revision = "90ed05df20d7"
+branch_labels = None
+depends_on = None
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("services_meta_data", "deprecated")
```

Comment:
did you forget to commit changes in packages/postgres-database/src/simcore_postgres_database/models/services.py ??

Reply:
sorry this was draft, thanks for the suggestion, changes are now added